Makefile

```
@@ -13,3 +13,7 @@ lint:
 
 tests:
 	pytest -vv
+
+arroyodocs:
```

```suggestion
docs:
```
I think it's a bit redundant to prefix make command with the name of the library